### ISSUE TITLE
frechet mean of spd matrices

### ISSUE THREAD

**Issue by loic001** [geomstats/geomstats#460]
Hi, first, thank you for this great library !

I would like to compute the mean of spd matrices.
Given this code :

```
from geomstats.geometry.spd_matrices import SPDMatrices, SPDMetricAffine

space = SPDMatrices(10)
data = space.random_uniform(n_samples=100)

metric = SPDMetricAffine(10)

mean = FrechetMean(metric=metric, method='default')
mean.fit(data)

mean_estimate = mean.estimate_
```

I get this error :
`ValueError: operand has more dimensions than subscripts given in einstein sum, but no '...' ellipsis provided to broadcast the extra dimensions.`

Could you please tell me what I'm doing wrong ?

Thanks !

**Comment by nguigs** [comment]
Hi! 

Thank you for your interest in `geomstats` !

There is indeed a bug when computing Frechet means on matrices. This is fixed in #462 which will be merged asap. Note that you will have to pass the argument `point_type='matrix'` when initializing the `FrechetMean` estimator.

**Comment by loic001** [comment]
Thank you😊
